### PR TITLE
.env has passed from command line and docker compose file has been ed…

### PR DESCRIPTION
**Pull request by kamolsavar**
…ited

Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.8 as base
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH /root/.poetry/bin:$PATH
+
+WORKDIR /DEVOPS-COURSE-STARTER
+
+# Install Dependencies
+COPY ./pyproject.toml .
```
Comment (hugh-emerson) [inline Dockerfile:10]: You should also copy in the `poetry.lock` file - this ties you to more specific versions of the dependencies (and nested dependencies). So you don't accidentally install something new unexpectedly.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+FROM python:3.8 as base
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH /root/.poetry/bin:$PATH
+
```
Comment (hugh-emerson) [inline Dockerfile:6]: You might want to set some other environment variables.

E.g. you could set `ENV FLASK_APP=todo_app/app`

And you could set `ENV FLASK_ENV=production` inside the production stage.

File: pyproject.toml [hunk 3]
```diff
@@ -11,6 +11,7 @@ python-dotenv = "^0.14.0"
 requests = "^2.25.1"
 pytest = "^6.2.4"
 selenium = "^3.141.0"
```
Comment (hugh-emerson) [inline pyproject.toml:13]: Poetry lets you set some dependencies as "dev only", meaning that they aren't needed when running your app in production. `pytest` and `selenium` are good examples of this.

You can move them down this file into the `dev-dependencies` section below, and then in your Dockerfile you could have different `poetry install` steps for the production and development stages. For production use `poetry install --no-dev`

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+version: "3.8"
```
Comment (hugh-emerson) [inline docker-compose.yml:1]: You should update your README.md to mention running the app in Docker and what the correct command is.

**Review by hugh-emerson — APPROVED** [review]
I've left some suggestions for possible improvements, but it looks good and you can merge when you're ready 👍

**Comment by kamolsavar** [comment]
Thank you Hugh for your  feedback
I will act on them

Regards

**Pull request merged**